web/src/integrationTest/kotlin/no/kartverket/matrikkel/bygning/v1/intern/egenregistrering/EgenregistreringRouteTest.kt

```diff
@@ -327,7 +323,7 @@ class EgenregistreringRouteTest : TestApplicationWithDb() {
             val response = client.post("/v1/intern/egenregistreringer") {
                 contentType(ContentType.Application.Json)
                 setBody(
-                    EgenregistreringRequest.validEgenregistrering(),
+                    EgenregistreringRequest.validBruksenhetRegistreringRequest(),
```

Comment:
Hva med noe ala dette? Tenker det med `bruksenhetRegistrering` ikke er så relevant her:

```suggestion
                    EgenregistreringRequest.gyldigRequest(), // eventuelt EgenregistreringRequest.validRequest(),
```